src/routes/signupRoutes.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
```

Comment:
The file name should be renamed to `auth` based on the feedback I gave during today's standup